geo-data-server/src/main/java/com/powsybl/geodata/server/GeoDataController.java

```diff
@@ -58,7 +54,6 @@
     public ResponseEntity<List<SubstationGeoData>> getSubstations(@RequestParam UUID networkUuid,
                                                                   @RequestParam(required = false) List<String> countries) {
         Set<Country> countrySet = toCountrySet(countries);
-        LOGGER.info("Loading substations geo data for countries {} of network '{}'", countrySet, networkUuid);
```

Comment:
Why did you remove these logs?

Reply:
to resolve a sonar issue, Sonar don't recommend to call a logger in an endpoint method.

Reply:
So move it to service side instead of removing it